### DISCUSSION TITLE
AOC Day 17 Discussion: Alexey Pajitnov's nightmare

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#783] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 17](https://adventofcode.com/2022/day/17) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
This was hard! I took a very data science-y approach to part 2 and applied a heuristic - it [worked](https://github.com/rhdeck/aoc-2022/blob/main/src/17/day_17.ipynb)! Also cheers to python for being so fast when I refactored into list comprehensions. next() is a powerhouse!

**Comment by doleraj** [comment]
https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day17/index.ts

☠️ ☠️ ☠️ ☠️ ☠️ ☠️ ☠️
